src/Components/FieldCheckbox/FieldCheckbox.js

```diff
@@ -73,35 +78,38 @@ const defaultProps = {
  */
 
 function FieldCheckbox({
-  id,
-  label,
-  isSelected,
-  isInvalid,
   className,
+  disabled,
   helpText,
   hideLabel,
-  value,
+  id,
+  isInvalid,
+  isSelected,
+  label,
   onChange,
-  disabled,
+  required,
   toggle,
   validationText,
+  value,
   ...rest
 }) {
+  const labelSpan = <span>{label}</span>;
```

Comment:
the more i look at this, the more I think these span/div wrappers should have a class like:
```suggestion
  const labelSpan = <span class="form-required">{label}</span>;
```
and then the css would target that instead of doing `.required > div > span:first-child::after...`
```suggestion
  const labelSpan = <span>{label}</span>;
```